.dockerignore

```diff
@@ -3,6 +3,7 @@ run
 plots
 .vscode
 .deps
+!requirements.txt
```

Comment:
is `requirements.txt` ignored by `docker` by-default? If not, is there a reason to add this?

Reply:
Right, so for activate-venv.sh and requirements.txt I encountered failures where those files could not be found (and are needed by install-build-tools.sh). So it was necessary to add the 'COPY' lines and this ! exception argument to the best of my understanding

Reply:
I think this should work without `requirements.txt` in `.dockerignore` but you are right an explicit `COPY` command is required in `Dockerfile` for `requirements.txt`

Reply:
Let me run without this line and see what happens
Edit: passes, good dd